### PR TITLE
Upgrade group entity to add description

### PR DESCRIPTION
**Pull request by ndufreche**
see backbee/BbCoreJs#765

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
tests broken :) please fix I need this ``removeUser`` too :p

**Comment by ndufreche** [comment]
Is why it's in WIP :p

File: Security/Group.php [hunk 1]
```diff
@@ -190,6 +224,34 @@ public function addUser(User $user)
...
+    }
+
+    /**
+     * @codeCoverageIgnore
+     *
+     * @param \BackBee\Security\User $user
+     *
+     * @return \BackBee\Security\Group
+     */
+    // public function removeUser(User $user)
```
Comment (ndufreche) [inline Security/Group.php]: I don't know can I had of this historical function. The good function to remove an entity from an Many To Many in doctrine is with "removeElement" function

Comment (mickaelandrieu) [inline Security/Group.php]: I agree with you, it's ``removeElement()`` function.

Comment (ndufreche) [inline Security/Group.php]: http://doctrine-orm.readthedocs.org/en/latest/reference/working-with-associations.html#removing-associations from the doc :D

Comment (ndufreche) [inline Security/Group.php]: Before an research in the Core php I don't found a usage of this function, I remove it.

**Comment by mickaelandrieu** [comment]
@ndufreche this need a squash :)

**Comment by ndufreche** [comment]
Just I watch the Travis validation.then(squash, resolve)

**Comment by mickaelandrieu** [comment]
:+1:

**Comment by eric-chau** [comment]
Review done with @ndufreche which agrees with the fact that we can improve the logic so I created an issue #428. We decide to merge it like this at moment cause it will imply too much change in PHP and JS sides. :+1:

**Pull request merged**